RIS.js

```diff
@@ -1451,6 +1452,19 @@ function applyValue(item, zField, value, rawLine) {
 				item.extra = value;
 			}
 		break;
+		case 'DOI':
+			if (!ZU.fieldIsValidForType("DOI", item.itemType) && ZU.cleanDOI(value)) {
+				value = ZU.cleanDOI(value);
+				if(item.extra) {
+					item.extra += '\nDOI: ' + value;
+				} else {
+					item.extra = 'DOI: ' + value;
+				}
+			}
+			else {
+				item[zField] = value;
+			}
+		break;
```

Comment:
Alternatively, you can delete the `else` above and just continue without a `break`, which means that for `DOI` all `default` things will be performed and (depending on the `itemType`) also some more, i.e. delete these lines
```js
			else {
				item[zField] = value;
			}
		break;
```